src/Export/VTK.jl

```diff
@@ -110,7 +110,7 @@ function WriteVTK.vtk_point_data(vtkfile, dh::MixedDofHandler, u::Vector, suffix
         for fh in dh.fieldhandlers
             # check if this fh contains this field, otherwise continue to the next
             field_pos = findfirst(i->i == name, getfieldnames(fh))
-            if field_pos == 0 && continue end
+            if field_pos === nothing && continue end
```

Comment:
Not related to your changes, but I found this difficult to read. Perhaps:
```suggestion
            field_pos === nothing && continue
```

Reply:
I've also been thinking if it would be better to replace this by some `hasfield(fh, fieldname)`.